apis/composition.yaml

```diff
@@ -22,6 +22,8 @@ spec:
             settings:
               - diskSize: 20
                 tier: db-f1-micro
+          writeConnectionSecretToRef:
+            namespace: upbound-system
```

Comment:
Glancing at the docs it says `name` is also required here: https://marketplace.upbound.io/providers/upbound/provider-gcp/v0.28.0/resources/sql.gcp.upbound.io/DatabaseInstance/v1beta1#doc:spec-writeConnectionSecretToRef

Reply:
@jastang yep! We are patching it below 👍🏻